plugins/faustwp/includes/graphql/callbacks.php

```diff
@@ -44,6 +45,28 @@ function register_templates_field() {
 	);
 }
 
+add_filter( 'graphql_get_setting_section_field_value', __NAMESPACE__ . '\\filter_introspection', 10, 5 );
+
+function filter_introspection( $value, $default_value, $option_name, $section_fields, $section_name ) {
```

Comment:
We should probably add a unit test for this function

Reply:
Added in https://github.com/wpengine/faustjs/pull/1594/commits/ceb2c5d8e07709ec76336cf24b394d11d9e94904